libs/wingsdk/src/util/util.ts

```diff
@@ -65,6 +104,127 @@ export interface Output {
   readonly status: number;
 }
 
+/**
+ * Handle to a running child process.
+ */
+export class ChildProcess {
+  /**
+   * @internal
+   */
+  public static _toInflightType(): string {
+    return InflightClient.forType(__filename, this.name);
+  }
+
+  /**
+   * The underlying native child process object representing the spawned process.
+   */
+  private childProcess: ReturnType<typeof spawn>;
+  /**
+   * The child's OS-assigned process ID.
+   */
+  public readonly pid: number | undefined;
+  /**
+   * The accumulated standard output from the child process.
+   */
+  private stdout: string = "";
```

Comment:
```suggestion
  private readonly stdout: string = "";
```

Reply:
Why `readonly`? How can you keep adding incoming data to the log if it's not a mutable object?